test/bokken_web/controllers/guardian_controller_test.exs

```diff
@@ -151,48 +118,12 @@ defmodule BokkenWeb.GuardianControllerTest do
   end
 
   defp new_guardian(_) do
-    guardian = %{
-      city: "Braga",
-      mobile: "+351915026743",
-      first_name: "Ana",
-      last_name: "Silva Costa"
-    }
-
-    user = %{
-      email: "ana@gmail.com",
-      password: "guardian123",
-      role: "guardian",
-      active: true
-    }
-
-    new_user = Accounts.create_user(user)
-    attrs = Map.put(guardian, :user_id, elem(new_user, 1).id)
-
-    {:ok, guardian} = Accounts.create_guardian(attrs)
-
+    guardian = insert(:guardian)
     %{guardian: guardian}
   end
 
   defp new_guardian_update(_) do
```

Comment:
This function is now equal to `new_guardian`. So that, you only need `new_guardian`.